operations/hpe_pdu/hpe_pdu_admin_procedures.md

```diff
@@ -0,0 +1,119 @@
+
+1. Use the `ssh` command from your local machine:
+    ```bash
+    > ssh -L 8443:{PDU_xname}:443 -N root@{ncn-m001_ip}
+    ```
+    Where `{PDU_xname}` is the xname of the PDU and `{ncn-m001_ip}` is the ip address of ncn-m001.\
+    You will need to enter the root password for ncn-m001.
+
+2. Connect to: [`https://localhost:8443`](https://localhost:8443) using a web browser.
+
+3. Log in with the `admin` username.  Enter the admin password.  If you have not changed the admin password, there will be a prompt to change the password.
+
+### HPE PDU Initial Set-up
+Use this procedure to do the following:
+* Ensure redfish is enabled
+* Add the default user
+* Enable Outlet Control
+
+1. Connect to the HPE PDU Web Interface (See [Connect to HPE PDU Web Interface](#connect-to-hpe-pdu-web-interface)) and log in as `admin`.
+
+#### Ensure Redfish is Enabled
+1. Using the **"Settings"** Icon (gear in computer monitor in top right corner) navigate to **"Network Settings"**.
+2. Verify there is a check next to "RESTapi Access", if not, click the **"Edit"** Icon (pencil) and enable.
+
+#### Add Default User
+1. Using the **"admin"** menu (top right corner) navigate to **"User Accounts"**.
+2. Click on the **"Add User"** button.
+3. Use the form to add the _username_ and _password_ for the default River user.  Assign the role _"Administrator"_ to that user.
+
+#### Enable Outlet Control
+1. Using the **"Home"** icon (House in top right corner) navigate to **"Control & Manage"**
+2. Verify **"Outlet Control Enable"** switch on the top of the page is selected (green)
+
+#### Update HPE PDU Firmware
+##### Check Firmware Version
+1. Connect to the HPE PDU Web Interface (See [Connect to HPE PDU Web Interface](#connect-to-hpe-pdu-web-interface)) and log in as `admin`
+2. Check which version of firmware is installed by selecting the **"Home"** icon (House in the top right corner) navigate to **"Identification"**
+3. The _"Version"_ will be displayed.  If the version is not _"2.0.0.L"_ update firmware.
+
+##### Update Firmware
+1. Download version **2.0.0.L** firmware from: [https://support.hpe.com/connect/s/search?language=en_US#q=P9S23A&t=All&sort=%40hpescuniversaldate%20descending&numberOfResults=25&f:@contenttype=[Drivers%20and%20Software]](https://support.hpe.com/connect/s/search?language=en_US#q=P9S23A&t=All&sort=%40hpescuniversaldate%20descending&numberOfResults=25&f:@contenttype=[Drivers%20and%20Software])
+This will download an ".exe" file which is a self extracting zip file.
+2. If using a windows system, run the .exe file to extract the files, or use an unzip program on the file.  One of the files extracted will be named **"HPE.FW"**, that is the firmware file you will need for uploading.
+3. Connect to the HPE PDU Web Interface (See [Connect to HPE PDU Web Interface](#connect-to-hpe-pdu-web-interface)) and log in as `admin`.
+4. Using the **"Settings"** Icon (gear in computer monitor in top right corner) navigate to **"System Management"**

```

Comment:
- Add period to the end of this step
- Icon can be lower case

Reply:
Updated